__tests__/app.test.js

```diff
@@ -186,5 +186,88 @@ describe('GET /api/articles/:article_id/comments', () => {
    });
 
 
+   test('POST 201: Responds with created comment upon successful POST', () => {
+    let commentToAdd = {
+      username: 'rogersop',
+      body: 'Foo bar'
+    }
+    return request(app)
+    .post('/api/articles/1/comments')
+    .send(commentToAdd)
+    .expect(201)
+    .then(( result ) => { 
+      expect(result.body.comment).toMatchObject({
+        author: 'rogersop',
+        body: 'Foo bar',
+        article_id: 1
+      })
+      expect(result.body.comment.hasOwnProperty("votes")).toBe(true);
+      expect(result.body.comment.hasOwnProperty("created_at")).toBe(true);
```

Comment:
These could be included in your toMatchObject. I recommend you to look into using expect.any() instead of the hardcoded value. (`{key: expect.any(Number)}`)